game_api.txt

```diff
@@ -792,7 +792,7 @@ Creates panes that automatically connect to each other
 		groups = {group = rating}, -- Uses the known node groups, see [Known damage and digging time defining groups]
 		sounds = SoundSpec,        -- See [#Default sounds]
 		recipe = {{"","","","","","","","",""}}, -- Recipe field only
-		use_texture_alpha = true, -- Optional boolean (default: `false`) for colored glass panes
+		use_texture_alpha = false, -- Optional boolean (default: `true`) for colored glass panes
```

Comment:
Surely this should be
`use_texture_alpha = true, ...`
As most xpanes have transparent pixels, in which case this field should be `true` ?
I also cannot see why coloured glass panes make a difference here, they should also have `true`? So i cannot see a need to mention them in a comment.
The boolean is not optional anymore, it should be set to `true` for any xpanes that contain transparent pixels.
So it seems a commrnt is not needed at all.
See https://github.com/minetest/minetest/pull/10122

Reply:
This boolean defaults to `true`, and should be set to `false` for opaque panes. So perhaps something like:
`use_texture_alpha = true, -- Set to 'false' for opaque panes, to increase rendering performance. Default is 'true'`
?

Reply:
> So perhaps something like:

Right. Will change that accordingly.